scaffolds/nuxt/scaffold.tsx

```diff
@@ -24,7 +24,6 @@ export default createScaffold<NuxtData>(
       <Template source="./nuxt.config.js" />
       <Template source="./package.json" />
       <Template source="./README.md" />
-      <Template source="./yarn.lock" />
```

Comment:
Why is yarn lock removed?

Reply:
My guess is that this was left in the template by mistake. Dependency lock files should not be rendered into the boilerplate code for JS projects because the user has the option to choose their NPM client (`npm` or `yarn`).